docs/how-to.md

```diff
@@ -161,19 +161,19 @@ Example: `dcld tx pki propose-add-x509-root-cert --certificate="----BEGIN CERTIF
 ### 6. Approve proposed X509 root certificate
 
 ```bash
-dcld tx pki approve-add-x509-root-cert --subject=<string> --subject-key-id=<hex string> --from=<account>
+dcld tx pki approve-add-x509-root-cert --subject=<string(Base64)> --subject-key-id=<hex string> --from=<account>
```

Comment:
`<string(Base64)>` is confusing. Make it `<base64 string>`

Reply:
Done! ✅